doc/manuals/installation_and_administration_manual/streaming/stream_processing_sinfonier_drawer.md

```diff
@@ -0,0 +1,65 @@
+#<a name="top"></a>Stream processing: Sinfonier Drawer
+
+Content:<br>
+
+* [Sinfonier Drawer](#section1)
+* [Requisites](#section2)
+* [Install](#section3)
+* [Start](#section4)
+* [Integrated](#section5)
+
+##<a name="section1"></a>Sinfonier Drawer
+
+FrontEnd part of Sinfonier Project. Allow users to define Apache Storm Topologies (DAG - Directed acyclic graph) in a visual way and send it to Storm Cluster using [Sifonier API](https://github.com/sinfonier-project/sinfonier-api)
+
+This project is a fork from webhookit [webhookit](http://neyric.github.com/webhookit)
+
+[Top](#top)
+
+##<a name="section2"></a>Requisites
+
+ * [Node.js](http://nodejs.org/) (>= 0.6.6)
+ * [npm](http://npmjs.org/)
+ * [mongoDB](http://www.mongodb.org/) (>= 2)
+
+[Top](#top)
+
+##<a name="section3"></a>Install
+
+    git clone https://github.com/sinfonier-project/sinfonier-drawer.git
+    cd sinfonier-drawer
+    npm install .
+
+[Top](#top)
+
+##<a name="section4"></a>Start
+
+* Ensure you have a MongoDB running on Localhost
+* Start Node Server
+ 
+```sh
+cd /path/to/sinfonier-drawer
+node app.js    
+```
+
+* Browse this URL [http://localhost:8124](http://localhost:8124)
```

Comment:
I would write "Brose this [URL](http://...), the create indexes on MongoDB and `supersinfonier` user".